### PR TITLE
Add support for request.get's 'params' param

### PR DESCRIPTION
**Pull request by alexjurkiewicz**
Requests defines the following API:
`get(url, params=None, **kwargs)`

### CODE REVIEW

File: third_party/2/requests/api.pyi [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Stubs for requests.api (Python 3)
 
-from typing import Union
+from typing import Union, Optional
 
 from .models import Response
 
 def request(method: str, url: str, **kwargs) -> Response: ...
-def get(url: Union[str, unicode], **kwargs) -> Response: ...
+def get(url: Union[str, unicode], params: Optional[dict]=None, **kwargs) -> Response: ...
```
Comment (gvanrossum) [inline third_party/2/requests/api.pyi]: - The docstring suggests that `params` could also be a `bytes`.
- Could you be more specific about the key and value types of the dict? Even`Dict[str, Any]` would be an improvement.

Comment (alexjurkiewicz) [inline third_party/2/requests/api.pyi]: Actually, re-reading https://docs.python.org/3/library/typing.html the `AnyStr` type doesn't allow type mixing, which this function allows, so I'll need to replace `AnyStr` with `Union[str, unicode]` (for python2 version) and `Union[str, bytes]` (for py3 version)...

**Comment by alexjurkiewicz** [comment]
I've added support for string form, and tightened the dict form type to allow only string keys/vals. 

As per my other comment the code will technically accept any types for dict keys/vals, but it seems better to be a little strict so the stub actually does something here.

File: third_party/2/requests/api.pyi [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
-from typing import Union
+from typing import Union, Optional, AnyStr
 
 from .models import Response
 
 def request(method: str, url: str, **kwargs) -> Response: ...
-def get(url: Union[str, unicode], **kwargs) -> Response: ...
+
+def get(url: Union[str, unicode],
+        params: Optional[Union[dict[Union[str, unicode],
```
Comment (ambv) [inline third_party/2/requests/api.pyi]: nit: `Optional[]` is redundant if your default value is `None`.

**Comment by ambv** [comment]
✨ 🍰 ✨  Thanks!

**Pull request merged**